rdmd.d

```diff
@@ -130,6 +131,7 @@ int main(string[] args)
             "eval", &eval,
             "loop", &loop,
             "exclude", &exclusions,
+            "include", (string opt, string p) { exclusions = exclusions.filter!(ex => ex != p).array(); },
```

Comment:
Ok I've confirmed it actually iterates over the options in order first, not the arguments. So the PR is ok.